### ISSUE TITLE
Citation rich static site generators

### ISSUE THREAD

**Issue by ashwinvis** [writing-resources/awesome-scientific-writing#6]
Candidates:

- https://github.com/inukshuk/jekyll-scholar
- https://github.com/mfenner/jekyll-pandoc

**Comment by maehr** [comment]
In my opinion, the HTML export options of most tools are sufficient for "simple" science blogging. More complex projects can be realized with a CMS. There are several resources that cover this topic: https://github.com/sindresorhus/awesome#content-management-systems

**Comment by ashwinvis** [comment]
True, but static site generators are fairly common nowadays and there are many around. Hosting a CMS requires one to host a server. I use Pelican for my website and to keep the [list of publications](https://ashwin.info.tm/pages/research) up to date I had used [pelican-cite](https://github.com/VorpalBlade/pelican-cite/) and now I use [bibbase service](https://bibbase.org/).

Blogging is also a form of writing isn't it? :pensive:

**Comment by maehr** [comment]
> True, but static site generators are fairly common nowadays and there are many around. Hosting a CMS requires one to host a server. I use Pelican for my website and to keep the [list of publications](https://ashwin.info.tm/pages/research) up to date I had used [pelican-cite](https://github.com/VorpalBlade/pelican-cite/) and now I use [bibbase service](https://bibbase.org/).
> 
> Blogging is also a form of writing isn't it? 😔

Yes of course, blogging is essential for science and we should embrace it. I just have not found an awesome way to do it until now. How is Pelican in regard to SEO?

**Comment by ashwinvis** [comment]
There is SEO is depends on the specific theme one uses since most of the metadata goes in the html headers. I think m.css - the theme that I use - does it by default. And here is another one:

https://elegant.oncrashreboot.com/search-engine-and-social-media-optimization

**Comment by maehr** [comment]
@ashwinvis Can this issue be closed?

**Comment by ashwinvis** [comment]
Considering there are tools like Quarto and MyST, this can be closed.
